doc/upgrading.md

```diff
@@ -15,6 +15,15 @@ See the [NEWS](../NEWS.md) summary of changes for each version.
 * [Version 25](#version-25)
 * [Version 20](#version-20)
 
+## Version 31
+
+* [RMB-328](https://issues.folio.org/browse/RMB-328) Update to OpenJDK 11.
+  A few updates to `pom.xml` are necessary. For `maven-compiler-plugin`
+  update `version` to `3.8.1`. and use `<release>11</release>` instead
+  of `source` and `target` elements.
+  Update aspectj version to 1.9.5. Update `aspect-maven-plugin` with
+  groupId `com.nickwongdev` and version `1.12.6`.
```

Comment:
Can you also mention to add `buildNode = 'jenkins-agent-java11'` to Jenkinsfile, change `folioci/alpine-jre-openjdk8:latest` to `folioci/alpine-jre-openjdk11:latest` in Dockerfile and to remove docker/docker-entrypoint.sh?

Reply:
What the problem with entrypoint.sh?

Reply:
Neither alpine-jre-openjdk8 nor alpine-jre-openjdk11 use it, and upgrading.md is missing a suggestion to remove it. This is a good opportunity to clean up.